tests/chunked-encode-large.rs

```diff
@@ -67,22 +68,23 @@ const TEXT: &'static str = concat![
 
 #[async_std::test]
 async fn chunked_large() -> Result<(), http_types::Error> {
-    let server = task::spawn(async {
+    let bind = test_utils::determine_port_to_bind().await;
```

Comment:
Can we rename the `bind` variable to `port`?

Reply:
would socket_addr work? I initially had `port` (as still remains in the function name, which will also be fixed), but it's both `localhost` as well as the port